courses/machine_learning/text_classification/automl_for_text_classification.ipynb

```diff
@@ -0,0 +1,499 @@
+{
```

Comment:
<p>typo: "..model we have just train." to "model we have just trained."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done!</p>